mmcls/models/backbones/vision_transformer.py

```diff
@@ -98,6 +101,109 @@ def forward(self, x):
         return x
 
 
+class BEiTTransformerEncoderLayer(TransformerEncoderLayer):
+    """Implements one encoder layer in Vision Transformer.
+
+    Args:
+        embed_dims (int): The feature dimension.
+        num_heads (int): Parallel attention heads.
+        feedforward_channels (int): The hidden dimension for FFNs.
+        drop_rate (float): Probability of an element to be zeroed
+            after the feed forward layer. Defaults to 0.
+        attn_drop_rate (float): The drop out rate for attention layer.
+            Default: 0.0.
+        drop_path_rate (float): Stochastic depth rate. Default 0.0.
+        num_fcs (int): The number of fully-connected layers for FFNs.
+            Default: 2.
+        bias (bool): The option to add leanable bias for q, k, v. If bias is
+            True, it will add leanable bias. If bias is 'qv_bias', it will only
+            add leanable bias for q, v. If bias is False, it will not add bias
+            for q, k, v. Default to 'qv_bias'.
+        act_cfg (dict): The activation config for FFNs.
+            Default: dict(type='GELU').
+        norm_cfg (dict): Config dict for normalization layer.
+            Default: dict(type='LN').
+        window_size (tuple[int], optional): The height and width of the window.
+            Default: None.
+        attn_cfg (dict): The configuration for the attention layer.
+            Defaults to dict().
+        ffn_cfg (dict): The configuration for the ffn layer.
+            Defaults to dict().
+        init_values (float, optional): Initialize the values of BEiTAttention
+            and FFN with learnable scaling. Default: None.
```

Comment:
Please rename the argument, `init_values` is confusing, maybe
```suggestion
        layer_scale_init_value (float, optional): The initialization value for
            the learnable scaling of attention and FFN. If None, xxxxx.
            Defaults to None.
```
And does it accepts None?